app/policies/user_policy.rb

```diff
@@ -22,27 +22,65 @@ def index?
   def edit_by_project_admin?
     project_admin? && record.portal_teacher
   end
+
   def update_by_project_admin?
     project_admin? && record.portal_teacher
   end
 
+  def project_admin_for_user?
+    return false unless record.respond_to? :cohorts
```

Comment:
It would be good to check to make sure that `user` is not nil here too. If a admin gets logged out and they are on the users listing and then click one of the buttons, we'll get a exception notification if we don't check for the nil user.